docs/_source/guides/llms/conceptual_guides/rm.md

```diff
@@ -192,21 +192,25 @@ triplets = []
 
 # Loop over all records in the dataset
 for record in feedback.records:
+    # Ensure that the record has responses
+    if record.responses is None or len(record.responses) == 0:
+        continue
 
     # Ensure the response has been submitted (not discarded)
-    if record['responses'][0]['status'] == 'submitted':
+    response = record.responses[0]
+    if response.status == 'submitted':
 
         # Get the preferred response index from the feedback
-        preferred_index = record['responses']['values']['response_ranking']
+        preferred_index = response.values['response_ranking'].value
```

Comment:
don't we normally use double quotes?

Reply:
Yes I usually do, just that didn't want to change too many things in the original code so I kept everything as-is besides the `pydantic.BaseModel` aka `rg.FeedbackRecord` usage instead